### ISSUE TITLE
Fails to precompile for Julia v1.4.1

### ISSUE THREAD

**Issue by alecokas** [JuliaGPU/OpenCL.jl#186]
Hi, I'm unable to precompile OpenCL, and I suspect the issue comes down to the Julia version since it seems that there is already an open issue regarding Julia v1 support (#165).
```
julia> versioninfo()
Julia Version 1.4.1
Commit 381693d3df* (2020-04-14 17:20 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, skylake)
Environment:
  JULIA_EDITOR = "/snap/code/33/usr/share/code/code"
  JULIA_NUM_THREADS =
```
After installing OpenCL, I try to precompile the package but I receive this error:
```julia> using OpenCL
[ Info: Precompiling OpenCL [08131aa3-fb12-5dee-8b74-c09406e224a2]
ERROR: LoadError: LoadError: AssertionError: libopencl != ""
Stacktrace:
 [1] top-level scope at /home/aleco/.julia/packages/OpenCL/yqbHv/src/api.jl:10
 [2] include(::Module, ::String) at ./Base.jl:377
 [3] include(::String) at /home/aleco/.julia/packages/OpenCL/yqbHv/src/OpenCL.jl:4
 [4] top-level scope at /home/aleco/.julia/packages/OpenCL/yqbHv/src/OpenCL.jl:25
 [5] include(::Module, ::String) at ./Base.jl:377
 [6] top-level scope at none:2
 [7] eval at ./boot.jl:331 [inlined]
 [8] eval(::Expr) at ./client.jl:449
 [9] top-level scope at ./none:3
in expression starting at /home/aleco/.julia/packages/OpenCL/yqbHv/src/api.jl:10
in expression starting at /home/aleco/.julia/packages/OpenCL/yqbHv/src/OpenCL.jl:25
ERROR: Failed to precompile OpenCL [08131aa3-fb12-5dee-8b74-c09406e224a2] to /home/aleco/.julia/compiled/v1.4/OpenCL/qfx5r_hvGgx.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1272
 [3] _require(::Base.PkgId) at ./loading.jl:1029
 [4] require(::Base.PkgId) at ./loading.jl:927
 [5] require(::Module, ::Symbol) at ./loading.jl:922
```

Is there any way for me to get around this problem? OpenCL is a dependency for another package I would like to use.

**Comment by jpsamaroo** [comment]
Would be fixed by #177

**Comment by alecokas** [comment]
Now successfully loads with the warning added by PR #177
